src/app/copilot/components/copilot-panel/copilot-panel.component.scss

```
@@ -393,6 +397,50 @@ $ease: cubic-bezier(0.4, 0, 0.2, 1);
+        margin: $u 0;
+        padding: ($u + 2px) ($u * 2);
+        border-radius: $r-sm;
+        max-width: 100%;
+        overflow-x: auto;
+        font-size: 12.5px;
+        line-height: 1.55;
+
+        code {
+          font-family: SFMono-Regular, Consolas, 'Liberation Mono', Menlo, monospace;
+          white-space: pre-wrap;
+          word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)